config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
I think we should remove this? We don't have action cable set up for this project and also don't need it.

Reply:
Don't we have to remove cable from the application.rb then as well?